### DISCUSSION TITLE
Exposing with the histogram

### DISCUSSION THREAD

**Discussion by sanj3k** [gopro/labs#277] [General]
Hi, I am not new to gopros and their settings etc but I have just started using the histogram especially with logb 400 and wide gamut it will be more difficult to expose correct by the eye I fear. I am familiar with the expose to the right technique and try to max it without clipping etc. but since more often its beneficial to lock the exposure settings like shutterspeed and iso for certain scenes then put on nd filters. I would be interested in your opinions what the histogram should look ideally in a from to limit. Say iso800 is the max but if possible the lowest iso setting would be prioritized first after that I set the shutterspeed to 180deg means 1/60 in a 30fps video and 1/120 in a 60fps one. 

I have also seen here that quite a lot use the ev-2 setting. Would it have any effect at all if I use min/max shutter at the same value ?

**Comment by sanj3k** [comment]
Also what does the dotted line stand for?

![image](https://user-images.githubusercontent.com/21160640/226565922-cacc6995-64a7-4c0e-aae4-4b7964ce86dc.png)

**Reply by dnewman-gpsw** [reply]
Mean/average.

**Comment by dnewman-gpsw** [comment]
EV-2 is used with LOGB=400, if you are using auto exposure, it will provide a good two stops of headroom in your highlights, with the log curve emphasizing shadow texture.   

If you are locking ISO and shutter, EV is not available (and would be meaningless, as there is no aperture control.)  

All your exposure plans are up to you, for your project, and your flying ability. i.e. 180 degree shutter is tricky from a drone, unless it is large and stable. This is why FPV users argue for 45 or 90 degree (for 30 fps.)  If you can keep the ISO lower, yes it is better.  Add ND until the ISO needs to go up.

**Reply by sanj3k** [reply]
I have heard now a couple of times that there are people not satisfied with 180deg shutter angle whats the exact explanation for that? At what should I pay attention? I think its just extremly motion blurry and at least with 24/25fps you really have to take slow turns because it starts to look choppy when turning to fast but this is kind of the og cinematic experience like in films

**Reply by dnewman-gpsw** [reply]
You are correct in using slow turns for cinematic look at 24/25. The issue with 180 degree for FPV, if you ever need a correction move, that will be stabilized away with Hypersmooth or in post, the move with add too much blur.  So its not about cinematic, is about electronic stabilization, which simply works better with a faster shutter. 

In this shot of mine (trying for cinematic moves) I'm using more like a 45 degree shutter average (I used Labs to limit max shutter to 90, but otherwise auto,) yet there is a nasty blur moment two seconds into the launch.  This blur would have been way worse at 180 degrees https://youtu.be/wsJKA-rr0Ko
